vowpalwabbit/vw_core.vcxproj

```diff
@@ -145,6 +145,7 @@
     <ClInclude Include="cb_algs.h" />
     <ClInclude Include="cb_dro.h" />
     <ClInclude Include="cb_explore_adf_bag.h" />
+    <ClInclude Include="cb_explore_adf_synthcover.h" />

```

Comment:
I think you also need to add the `.cc` file (after line 270 in this file)

Reply:
thanks!